sohojogi/pubspec.yaml

```
@@ -78,8 +79,16 @@ flutter:
   # "family" key with the font family name, and a "fonts" key with a
   # list giving the asset and other descriptors for the font. For
   # example:
-  # fonts:
-  #   - family: Schyler
+  fonts:
+     - family: Poppins
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 5

(indentation)